docs/src/pages/en/media-live-button.md

```diff
@@ -0,0 +1,115 @@
+---
+title: <media-live-button>
+description: Media Live Button
+layout: ../../layouts/MainLayout.astro
+source: https://github.com/muxinc/media-chrome/tree/main/src/js/media-live-button.js
+---
+
+The Media Live Button shows when the stream is live via an indicator (red dot, by default). It also allows the viewer to seek to the most current part of the stream by clicking on the button ("seek to live").
+
+<h3>Default: Media is not live</h3>
+
+<media-live-button></media-live-button>
+
+```html
+<media-live-button></media-live-button>
+```
+
+<h3>Media is live</h3>
+
+<media-live-button media-time-is-live></media-live-button>
+
+```html
+<media-live-button media-time-is-live></media-live-button>
+```
+
+<h3>Alternate text</h3>
+
+<media-live-button media-time-is-live>
+  <span slot="text">Hello!</span>
+</media-live-button>
+
+```html
+<media-live-button>
+  <span slot="text">Hello</span>
+</media-pip-button>
+```
+
+<h3>Alternate indicator SVG</h3>
+
+<media-live-button media-time-is-live>
+  <svg slot="indicator" viewBox="0 0 16 16"><circle cx="8" cy="8" r="8"></circle></svg>
+</media-live-button>
+
+```html
+<media-live-button media-time-is-live>
+  <svg slot="indicator" viewBox="0 0 16 16"><circle cx="8" cy="8" r="8"></circle></svg>
+</media-live-button>
+```
+
+<h3>Alternate indicator text or font icon</h3>
+
+<media-live-button media-time-is-live>
+  <span slot="indicator">Hello!</span>
+</media-live-button>
+
+```html
+<media-live-button media-time-is-live>
+  <span slot="indicator">Hello!</span>
+</media-live-button>
+```
+
+<h3>Common Use: Text is also the indicator</h3>
+
+<style>
+  #textlive {
+    --media-live-indicator-off-icon-color: white;
+  }
+</style>
+<media-live-button id="textlive">
+  <span slot="indicator">LIVE</span>
+  <span slot="spacer"></span>
+  <span slot="text"></span>
+</media-live-button>
+<media-live-button id="textlive" media-time-is-live>
+  <span slot="indicator">LIVE</span>
+  <span slot="spacer"></span>
+  <span slot="text"></span>
+</media-live-button>
+
+```html
+<style>
+  #textlive {
+    --media-live-indicator-off-icon-color: white;
+  }
+</style>
+<media-live-button id="textlive">
+  <span slot="indicator">LIVE</span>
+  <span slot="spacer"></span>
+  <span slot="text"></span>
+</media-live-button>
+```
+
+## Attributes
+
+| Name            | Type      | Default Value | Description |
+| --------------- | --------- | ------------- | ----------- |
+| `media-time-is-live` | `boolean` | `false`| Include when the media time is at or close to the most current time or _live edge_. Clicking the button will not seek to live when true. [_Set automatically by media-controller._] |
+| `media-paused` | `boolean` | `false`| Include when the media is paused. The button will unpause the video if paused when seeking to live. [_Set automatically by media-controller._] |
```

Comment:
```suggestion
_None_
```
Currently, we only document attributes intended to be set by an implementor for customization/configuration (and not attributes set by the Media Controller). See, e.g. [media-play-button.md](https://github.com/muxinc/media-chrome/blob/main/docs/src/pages/en/media-play-button.md). If we'd like to document both (ideally with some differentiation between those two with an eye towards devex), we should plan this and do it as a single effort.

Reply:
Yeah, I saw that, and I disagree with only documenting the non-media-controller attributes, based on the goal of [element independence](https://github.com/muxinc/media-chrome/blob/main/ARCHITECTURE.md#element-independence). I also don't love requiring a big project to ship it all at once since the temporary inconsistency doesn't really hurt anyone and it's no small effort to document all the detail per attr per element. But I agree in that I think we should make it a goal of 1.0 to document those attrs everywhere so the inconsistency has a clear end date, and in that we can create some better visual separation between them.

Reply:
I do worry that having this one place where we document the "internal" attributes will be confusing in the interim and increases the chance of mistakes when we **_do_** update the docs to include both. I'd still recommend that we don't document currently (there's already some implicit documentation of those attributes above in terms how how they update the visual state), but I won't treat this as a blocker.